src/performance/metrics/PerformanceUtil.cpp

```diff
@@ -17,12 +17,14 @@ using namespace std::chrono;
 std::map<std::string, std::vector<ResourceUsageInfo>> resourceUsageMap;
 
 Logger scheduler_logger;
-SQLiteDBInterface sqlLiteDB = *new SQLiteDBInterface();
-PerformanceSQLiteDBInterface perfDb = *new PerformanceSQLiteDBInterface();
+SQLiteDBInterface *sqlLiteDB;
+PerformanceSQLiteDBInterface *perfDb;
 
 void PerformanceUtil::init() {
-    sqlLiteDB.init();
-    perfDb.init();
+    sqlLiteDB = new SQLiteDBInterface();
+    perfDb = new PerformanceSQLiteDBInterface();
```

Comment:
If `PerformanceUtil::init()` is called multiple times, won't this cause a memory leak?